flow-libs/pirates.js.flow

```diff
@@ -0,0 +1,17 @@
+// @flow
+
+// Derived from : https://github.com/ariporad/pirates/blob/master/index.d.ts
+// Which is licensed MIT.
+
+interface Options {
```

Comment:
Flow interfaces allow passing other (unlisted) properties. So this should probably be `type Options = {| `

https://flow.org/try/#0JYOwLgpgTgZghgYwgAgPIAczAPYgM7IDeAUMshAB5h4D8AXMgIJRRwCeAPHmFKAOYA+ANylkwPiGxQIAOWwATCAFkFAVwA2EWgwBG2bJrggRAX2LEYqkAiy5kMABTZMeBhlv4AlETMWHhPGwAWwgwAAt+AFF1PAgGAG0AXRNPISA